node-module-system/calculate.js

```diff
@@ -0,0 +1,14 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+if (process.argv.includes('plus')) {
```

Comment:
Using `includes` seems clever, but it allows strange things, like:
`node calculate plus 3 4`
which would actually print the wrong result.